internal/utils/cloudbuild.go

```diff
@@ -61,7 +61,7 @@ func BuildStepsToDAG(steps []*cloudbuild.BuildStep) *cgraph.Graph {
 
 	var nodeList []*cgraph.Node
 	for idx, step := range steps {
-		name := step.Id
+		name := step.Name
```

Comment:
The name of the step means the image being used in this step. It doesn't contain the information what this step will do. 
So, I don't think it is good to label the node with the name. We can have more labels to show these information on the node instead.

Reply:
Makes sense, I think my choice of example just sucked